CrewChiefV4/App.config

```diff
@@ -100,14 +100,11 @@
       <setting name="custom_device_guid" serializeAs="String">
         <value />
       </setting>
-      <setting name="r3e_launch_params" serializeAs="String">
+      <setting name="r3e_steam_id" serializeAs="String">
         <value>-applaunch 211500</value>
       </setting>
-      <setting name="r3e_launch_exe" serializeAs="String">
-        <value>c:/games/steam/steam.exe</value>
-      </setting>
       <setting name="launch_raceroom" serializeAs="String">
-        <value>False</value>
+        <value>True</value>
```

Comment:
This must have gotten toggled by accident when I was playing with the other settings. I'll change it back to 'False'.